modules/tftp/server.rb

```diff
@@ -7,33 +7,35 @@ class Server
     # Creates TFTP pxeconfig file
     def set mac, config
       raise "Invalid parameters received" if mac.nil? || config.nil?
-
       FileUtils.mkdir_p pxeconfig_dir
-
-      File.open(pxeconfig_file(mac), 'w') {|f| f.write(config) }
-      logger.info "TFTP: entry for #{mac} created successfully"
+      pxeconfig_file(mac).each do |file|
+        File.open(file, 'w') {|f| f.write(config) }
+        logger.info "TFTP: #{file} created successfully"
+      end
     end
 
     # Removes pxeconfig files
     def del mac
-      file = pxeconfig_file(mac)
-      if File.exist?(file)
-        FileUtils.rm_f file
-        logger.debug "TFTP: entry for #{mac} removed successfully"
-      else
-        logger.info "TFTP: Skipping a request to delete a file which doesn't exists"
+      pxeconfig_file(mac).each do |file|
+        if File.exist?(file)
+          FileUtils.rm_f file
+          logger.info "TFTP: #{file} removed successfully"
+        else
+          logger.warn "TFTP: Skipping a request to delete a file which doesn't exists"
```

Comment:
It doesn't look like this has any consequences for/may require an action from the user. I don't think we should log stuff at the level above "debug" ("info" for settings on startup is an exception) unless some action may be required (the level denotes the urgency of such an action).